examples/main_refactor.py

```diff
@@ -0,0 +1,165 @@
+    TVB, DATA_MODE
+from tvb_epilepsy.base.disease_hypothesis import DiseaseHypothesis
+from tvb_epilepsy.base.equilibrum_service import EquilibrumComputationService
+from tvb_epilepsy.base.plot_tools import plot_nullclines_eq, plot_sim_results
+from tvb_epilepsy.base.utils import initialize_logger, set_time_scales, calculate_projection, filter_data
+from tvb_epilepsy.custom.read_write import write_h5_model, write_ts_epi, write_ts_seeg_epi
+from tvb_epilepsy.tvb_api.epileptor_models import EpileptorDP
+
+if DATA_MODE is TVB:
+    from tvb_epilepsy.tvb_api.readers_tvb import TVBReader as Reader
+else:
+    from tvb_epilepsy.custom.readers_custom import CustomReader as Reader
+
+if SIMULATION_MODE is TVB:
+    from tvb_epilepsy.tvb_api.simulator_tvb import setup_simulation
+else:
+    from tvb_epilepsy.custom.simulator_custom import setup_simulation
+
+if __name__ == "__main__":
+    logger = initialize_logger(__name__)
+    data_folder = '/WORK/Episense/trunk/demo-data/Head_TREC'
+
+    reader = Reader()
+
+    logger.info("Reading from: " + data_folder)
+    head = reader.read_head(data_folder)
+
+    x0_indices = [20]
+    x0_values = numpy.random.normal(0.85, 0.02, (len(x0_indices),))
+
+    hypothesis = DiseaseHypothesis("x0", head.connectivity, x0_values, x0_indices, [], [], [], "x0_Hypothesis")
+
+    all_regions_one = numpy.ones((hypothesis.get_number_of_regions(),), dtype=numpy.float32)
+
+    epileptor_model = EpileptorDP(Iext1=3.1 * all_regions_one, yc=all_regions_one,
+                                  K=10 * all_regions_one / hypothesis.get_number_of_regions())
+
+    equilibrum_service = EquilibrumComputationService(hypothesis, epileptor_model)
+
+    model_configuration, lsa_hypothesis = equilibrum_service.configure_model_from_x0_hypothesis(None)
+
+    write_h5_model(hypothesis.prepare_for_h5(), folder_name=data_folder, file_name=hypothesis.get_name() + ".h5")
+    write_h5_model(lsa_hypothesis.prepare_for_h5(), folder_name=data_folder,
+                   file_name=lsa_hypothesis.get_name() + ".h5")
+
+    write_h5_model(model_configuration.prepare_for_h5(), folder_name=data_folder, file_name="Config.h5")
+
+    # plot_hypothesis_equilibrium_and_lsa(lsa_hypothesis, model_configuration, figure_dir=data_folder)
+
+    fs = 2 * 4096.0
+    scale_time = 2.0
+    time_length = 3000.0
+    scale_fsavg = 2.0
+    report_every_n_monitor_steps = 10.0
+    (dt, fsAVG, sim_length, monitor_period, n_report_blocks) = set_time_scales(fs=fs, dt=None,
+                                                                               time_length=time_length,
+                                                                               scale_time=scale_time,
+                                                                               scale_fsavg=scale_fsavg,
+                                                                               report_every_n_monitor_steps=report_every_n_monitor_steps)
```

Comment:
this line doesn't fit in the github review window.. way too long.  please adhere to PEP8: PyCharm supports it, and anyone reading your code expects it.